pkg/collector/py/check.go

```diff
@@ -89,6 +88,34 @@ func (c *PythonCheck) Run() error {
 	return errors.New(resultStr)
 }
 
+// RunSimple runs a Python check without sending data to the aggregator
+func (c *PythonCheck) RunSimple() error {
```

Comment:
Agreed, it would definitely be ideal to mock the aggregator... originally, the aggregator was being initialized here https://github.com/DataDog/datadog-agent/blob/isabelle/workers/pkg/collector/runner/num_workers_test.go#L127
but @masci suggested that was one of the problems contributing to the python checks crashing so I removed it... not sure how else to handle this without changing the Run interface...